src/many/Cargo.toml

```diff
@@ -47,6 +47,8 @@ sha3 = "0.9.1"
 simple_asn1 = "0.5.4"
 signature = "1.3"
 static_assertions = "1.1.0"
+strum = "0.24.0"
```

Comment:
Is this used anywhere?

Reply:
https://github.com/liftedinit/many-rs/pull/95/files#diff-a308dae0adf7823b4efd6fe570f06aaed4f246521bcbb4b15ffd2b5f1c2438c4R19-R21

Reply:
I think you might have missed a file.

Reply:
I did miss a file. Still lgtm!